modules/oe_whitelabel_helper/src/Plugin/Block/CorporateEuLogoBlock.php

```diff
@@ -83,19 +84,21 @@ public function build(): array {
     $image = [
       '#theme' => 'image',
       '#uri' => $logo_path . '/logo-eu--' . $language . '.svg',
-      '#width' => '240px',
-      '#height' => '60px',
       '#alt' => $title,
       '#title' => $title,
     ];
-    $mobile_url = file_create_url($logo_path . '/mobile/logo-eu--' . $language . '.svg');
```

Comment:
removing this is not good, see https://oelibrary.netlify.app/default/?path=/story/compositions-header--header&args=light:false, the EU logo has a mobile variant, that is what the picture/source setup is for, that is how browsers know to switch the image when in mobile